features/providers/check_nonpassport_autogrant.feature

```diff
@@ -82,7 +82,6 @@ Feature: Checking ccms means does NOT auto grant for non passported applications
 
 @javascript @vcr
   Scenario: I am able to create a non passported application without Income or Cap Contribs and with no restrictions
-    Given the setting to manually review all cases is enabled
     And The means questions have been answered by the applicant
```

Comment:
@mikekeen1 I'm going to be really picky and suggest that as you've removed the `Given` line, you should change `And The means questions have been answered by the applicant` to `Given The means questions have been answered by the applicant, so we maintain the Given-When-Then format.

Reply:
Ok, 3rd time lucky :-)